manifests/database/postgresql.pp

```diff
@@ -8,8 +8,8 @@
   $manage_server        = $puppetdb::params::manage_dbserver,
   $manage_package_repo  = $puppetdb::params::manage_pg_repo,
   $postgres_version     = $puppetdb::params::postgres_version,
-) inherits puppetdb::params {
-
+) {
+  include puppetdb::params
```

Comment:
Hmm, what is the value gained by changing this pattern to one using include? The style guide says this particular case is okay fwiw: https://docs.puppetlabs.com/guides/style_guide.html#class-inheritance. I think also this might be the reason some of the unit tests are failing.